CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [v1.3.0] - 2020-08-13
+### Changed
+- Do not disable Redis API when we are destroying instance of this module, because we can have more instances which needs it enabled
```

Comment:
Tbh, should have done it before, the use case is pretty obvious, can we shorten the description to maybe just: `Do not disable Redis API when the module's instance is destroyed`?

Reply:
We can, but we forget why we did it :) If you will ask me why I did that before on our modules, I will not remember :)

Reply:
done :)